src/main/java/frc/robot/shooter/ShootAndSpinUp.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.shooter;
+
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class ShootAndSpinUp extends SequentialCommandGroup {
```

Comment:
renamed, also quick question: should I be responding to every comment? on one hand it provides a record that the issue was addressed but otherwise it seems annoying or excessive.

Reply:
@Masbender you do not need to respond, we will see an "outdated" when you code on that line changed meaning you changed it. At work i usually add a thumbs up emoji react to keep track of which ones i have done and as kind of a "i did this". Responding to ones where you disagree or have other questions or clarification is normal though